sdk/src/trace/span.cc

```diff
@@ -136,6 +143,34 @@ void Span::AddEvent(nostd::string_view name,
   (void)attributes;
 }
 
+void Span::AddLink(const trace_api::Link &link) noexcept
+{
```

Comment:
Declare `lock_guard(mf_)` to make sure it is thread-safe?

Reply:
Have removed AddLink() as part of separate PR #351 . Please review that. thanks.